homeassistant/components/climate/mill.py

```diff
@@ -75,6 +75,18 @@ def unique_id(self):
         """Return a unique ID."""
         return self._heater.device_id
 
+    @property
+    def device_info(self):
```

Comment:
This requires use of config entries to be useful:
https://developers.home-assistant.io/docs/en/device_registry_index.html#defining-devices

Reply:
Ahh, I see